src/chanjo2/endpoints/overview.py

```diff
@@ -0,0 +1,34 @@
+from os import path
+
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from chanjo2.demo import DEMO_OVERVIEW_QUERY_DATA
+from chanjo2.meta.handle_overview_content import get_overview_data
+from chanjo2.models.pydantic_models import GeneralReportQuery
+
+
+def get_templates_path() -> str:
+    """Returns the absolute path to the templates folder of this app."""
+    APP_ROOT: str = path.abspath(path.join(path.dirname(__file__), ".."))
+    return path.join(APP_ROOT, "templates")
+
+
+templates: Jinja2Templates = Jinja2Templates(directory=get_templates_path())
+router = APIRouter()
+
+
+@router.get("/overview/demo", response_class=HTMLResponse)
+async def demo_overview(request: Request):
+    """Return a demo genes overview page over a list of genes for a list of samples."""
+
+    overview_query = GeneralReportQuery(**DEMO_OVERVIEW_QUERY_DATA)
+    overview_content: Dict = get_overview_data(query=overview_query)
```

Comment:
```suggestion
    overview_content: dict = get_overview_data(query=overview_query)
```
or extend the type hint.

Reply:
Yes but all the repo is in the other way now!

Reply:
I do not understand what you mean. How does it affect using a built in type hint? Do you mean this will be changed soon?

Reply:
Nono, I mean that we have Dict all over the rest of the code in this repo, but I guess it's fine to have dict whenere we don't have to specify the keys of the dict

Reply:
Aha, ok, then I understand. From python 3.9 you can subscripts are natively  supported and you can write dict[] instead having to import and write Dict[].